test/integration/helper.js

```diff
@@ -29,6 +29,29 @@
 
 'use strict';
 
+function strongPasswordRequired() {
+  // OpenSearch 2.12.X and later require strong passwords
+  const version = process.env.OPENSEARCH_VERSION;
+  if (!version || version === 'latest') return true;
+  const [major, minor, _] = version.split('.').map((v) => parseInt(v, 10));
+  if (major > 2) return true;
+  return major === 2 && minor >= 12;
+}
+
+function createSecuredClient() {
```

Comment:
👏

Reply:
where would this be used?

Reply:
This is meant to be used in integration tests only. It's a helper function inside `test` folder.